File: data/gpt-4o/readable_format_HPEC/fortran/file018.f90
program DRB087_static_data_member2_orig_yes
    use omp_lib
    use DRB087
    implicit none

    type(A) :: c
    c = A(0,0)

    !$omp parallel
    counter = counter+1
    pcounter = pcounter+1
    !$omp end parallel

    print *,counter,pcounter
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file018.cpp
#include <stdio.h>
#include <omp.h>
class A {
public:
    static int counter;
    static int pcounter;
    A(int a, int b) { counter=a; pcounter=b; }
};
int A::counter = 0;
int A::pcounter = 0;
int main()
{
#pragma omp parallel
{
    A::counter=A::counter+1;
    A::pcounter=A::pcounter+1;
}
printf("%d %d\n", A::counter, A::pcounter);
return 0;
}
